recipe/meta.yaml

```diff
@@ -1,47 +1,55 @@
+{% set version = "2.2.21" %}
+
 package:
-    name: udunits2
-    version: 2.2.20
+  name: udunits2
+  version: {{ version }}
 
 source:
-    fn: v2.2.20.tar.gz
-    url: https://github.com/Unidata/UDUNITS-2/archive/v2.2.20.tar.gz
-    sha1: a0d30024af16749a4a5ca60133b32735ae52f9e2
+  fn: v{{ version }}.tar.gz
```

Comment:
The `fn` does not matter as long as it has something unique, like the version number. We can use any name here. I like the `v` because I dislike file names that starts with numbers :wink: